builtins/web/fetch/headers.cpp

```diff
@@ -537,24 +602,24 @@ JSObject *Headers::create(JSContext *cx, host_api::HttpHeadersReadOnly *handle,
   return self;
 }
 
-JSObject *Headers::create(JSContext *cx, HandleValue init_headers, host_api::HttpHeadersGuard guard) {
+JSObject *Headers::create(JSContext *cx, HandleValue init_headers, HeadersGuard guard) {
   RootedObject self(cx, create(cx, guard));
   if (!self) {
     return nullptr;
   }
   if (!init_entries(cx, self, init_headers)) {
     return nullptr;
   }
-  MOZ_ASSERT(mode(self) == Headers::Mode::ContentOnly || mode(self) == Headers::Mode::Uninitialized);
+  MOZ_ASSERT(mode(self) == Headers::Mode::ContentOnly ||
+             mode(self) == Headers::Mode::Uninitialized);
   return self;
 }
 
 bool Headers::init_entries(JSContext *cx, HandleObject self, HandleValue initv) {
-  // TODO: check if initv is a Headers instance and clone its handle if so.
```

Comment:
Sure, readded - but just to note that Fastly doesn't have the capability to do headers handle cloning, so this would ideally be implemented in a way that can be turned off.